src/server/auth/auth-config.ts

```diff
@@ -47,10 +47,13 @@ export const oidcConfigBase: ConfigParams = {
     rolling: true,
     rollingDuration: OIDC_SESSION_MAX_AGE_SECONDS,
     name: OIDC_SESSION_COOKIE_NAME,
-    store: getSessionStore(openIdAuth as typeof expressSession, {
-      tableName: OIDC_SESSIONS_TABLE_NAME,
-      ttlSeconds: OIDC_SESSION_MAX_AGE_SECONDS, // NOTE: Not sure if this works with rollingDuration
-    }),
+    store:
+      getFromEnv('MA_APP_MODE') !== 'unittest'
+        ? getSessionStore(openIdAuth as typeof expressSession, {
+            tableName: OIDC_SESSIONS_TABLE_NAME,
+            ttlSeconds: OIDC_SESSION_MAX_AGE_SECONDS, // NOTE: Not sure if this works with rollingDuration
+          })
+        : undefined,
```

Comment:
Misschien beetje een nit, maar leest net iets makkelijker in zijn positieve vind ik.
`getFromEnv('MA_APP_MODE') === 'unittest' ? undefined : ...`